channels/channel_state.go

```diff
@@ -15,6 +15,8 @@ import (
 
 // channelState is immutable channel data plus mutable state
 type channelState struct {
+	// peerId of the manager peer
```

Comment:
this is an interesting change and probably useful. 

I dunno if you saw but there's a method called `OtherParty` that takes a peer.ID for the party the data transfer is running on.

For symmetry, I'd like to do two things:
I'd like to rename ManagerPeer to ThisParty or SelfParty
and then I'd like to remove the parameter to OtherParty since we now have it in the channel state.
I guess I'd also be fine with SelfPeer and OtherPeer if Party doesn't make sense.

I'm ok with doing this as a seperate cleanup PR btw as well.